Specs/Data/Cesium3DTiles/Metadata/ImplicitContentMetadata/subtrees/0.0.0.json

```diff
@@ -52,10 +52,10 @@
       "class": "building",
```

Comment:
We might have other changes to subtree files, so I should split the binary files into JSON like this one.

Reply:
Just pushed a commit for this. I'll do the other implicit tilesets but in a separate PR.

Reply:
opened https://github.com/CesiumGS/cesium/pull/10156 for this, it can be reviewed after this one.